sdk/communication/azure-communication-callingserver/README.md

```diff
@@ -0,0 +1,188 @@
+
+## Examples
+
+### Create CallClient
+
+Once you initialized a `CallClient` class, you can do the following server calling operations:
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L28-L34 -->
+```java
+// Your connectionString retrieved from your Azure Communication Service
+String connectionString = "endpoint=https://<resource-name>.communication.azure.com/;accesskey=<access-key>";
+
+// Initialize the calling server client
+final CallingServerClientBuilder builder = new CallingServerClientBuilder();
+builder.connectionString(connectionString);
+CallingServerClient callingServerClient = builder.buildClient();
+```
+
+### Create, AddParticipant, Hangup and Delete a call
+
+Create a Call: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L46-L66 -->
+```java
+CommunicationIdentifier source = new CommunicationUserIdentifier("<acs-user-identity>");
+CommunicationIdentifier firstCallee = new CommunicationUserIdentifier("<acs-user-identity-1>");
+CommunicationIdentifier secondCallee = new CommunicationUserIdentifier("<acs-user-identity-2>");
+
+CommunicationIdentifier[] targets = new CommunicationIdentifier[] { firstCallee, secondCallee };
+
+String callbackUri = "<callback-uri-for-notification>";
+
+CallModality[] requestedMediaTypes = new CallModality[] { CallModality.AUDIO, CallModality.VIDEO };
+
+EventSubscriptionType[] requestedCallEvents = new EventSubscriptionType[] {
+        EventSubscriptionType.DTMF_RECEIVED,
+        EventSubscriptionType.PARTICIPANTS_UPDATED };
+
+CreateCallOptions createCallOptions = new CreateCallOptions(
+    callbackUri,
+    requestedMediaTypes,
+    requestedCallEvents);
+
+CallConnection callConnection = callingServerClient.createCallConnection(source, targets, createCallOptions);
+```
+
+Add a participant to a Call:
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L86-L87 -->
+```java
+CommunicationIdentifier thirdCallee = new CommunicationUserIdentifier("<acs-user-identity-2>");
+callConnection.addParticipant(thirdCallee, "ACS User 2", "<string-for-tracing-responses>");
+```
+
+Hangup a Call:
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L76-L76 -->
+```java
+callConnection.hangup();
+```
+
+### Start, Stop, Pause, Resume, and Get a recording
+
+Create a ConverationClient: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ConversationClientReadmeSamples.java#L29-L35 -->
+```java
+// Your connectionString retrieved from your Azure Communication Service
+String connectionString = "endpoint=https://<resource-name>.communication.azure.com/;accesskey=<access-key>";
+
+// Initialize the calling server client
+final CallingServerClientBuilder builder = new CallingServerClientBuilder();
+builder.endpoint(endpoint).connectionString(connectionString);
```

Comment:
You don't need both the endpoint and connection string. 
It is either: 
1. Connection string
2. Endpoint and access key

Reply:
In addition, all the sample code for building the client should go into its own `Authentication` section. There is no need to repeat it throughout your samples for calling the client functions.

Reply:
builder is authenticated with connection string only and put in Authentication section